ui/StatusQ/src/onboarding/enums.h

```diff
@@ -22,7 +22,10 @@ class OnboardingEnums
 
         LoginWithSeedphrase,
         LoginWithSyncing,
-        LoginWithKeycard
+        LoginWithKeycard,
+
+        LoginWithLostKeycardSeedphrase,
```

Comment:
Do we need those? In the end, it's just a regular login with a keycard

Reply:
But yeah, keep them, and use them in `function test_loginScreen_launchesExternalFlow(data)` ;)